examples/jsm/controls/OrbitControls.js

```diff
@@ -514,6 +581,14 @@ class OrbitControls extends EventDispatcher {
 
 		}
 
+		function updateMouseParameters( event ) {
+
+			mouse.x = ( event.clientX / window.innerWidth ) * 2 - 1;
+			mouse.y = - ( event.clientY / window.innerHeight ) * 2 + 1;
+			dollyDirection.set( mouse.x, mouse.y, 1 ).unproject( object ).sub( object.position ).normalize();
```

Comment:
Also presumably this shouldn't assume that the full screen is used for controls. Is there a preferred way to get the dom element offset and relative cursor coord?